src/collections/events/kcd-chennai-2023/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: "KCD Chennai 2023"
+date: 2023-07-22
+thumbnail: ./kcd_chennai.webp
+darkthumbnail: ./kcd_chennai.webp
+eurl: https://www.kcdchennai.in/
+type: Event
+published: true
+product: Meshery
```

Comment:
Could you toss in "MeshMap" in addition to "Meshery"?

Reply:
@leecalcote  check out the changes in the latest commit https://github.com/layer5io/layer5/commit/0456d2cb5c40119c9dd2fef523b66708ee251bd5